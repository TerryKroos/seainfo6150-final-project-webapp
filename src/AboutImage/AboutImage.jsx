import React from 'react';
import styles from "./AboutImage.module.css";

const AboutImage = () => {
    return (
      <div className={styles.container}>
         <h1>About Us</h1>
      </div>
    );
  };

  export default AboutImage;