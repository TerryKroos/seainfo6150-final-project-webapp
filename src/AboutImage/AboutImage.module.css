.container {
    background-image: url(https://static.clubs.nfl.com/image/upload/v1585250148/vikings/at52xgdsujviaujyki4r.jpg);
    background-size: 100% 100%;
    height: 30em;
    display: flex;
    justify-content: center;
    align-items: center;
    color: black;
}