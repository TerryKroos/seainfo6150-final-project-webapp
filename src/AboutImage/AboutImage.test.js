
import React from "react";
import AboutImage from "./AboutImage";
import { render } from "@testing-library/react";

describe("AboutImage tests", () => {
    it("renders correctly", () => {
      const { container } = render(<AboutImage />);
      expect(container).toMatchSnapshot();
    });
  
  });