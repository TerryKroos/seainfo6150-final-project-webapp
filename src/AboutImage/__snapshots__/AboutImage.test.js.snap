// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutImage tests renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      About Us
    </h1>
  </div>
</div>
`;
